### DISCUSSION TITLE
Required field indicator customization (color, text, and hidden)

### DISCUSSION THREAD

**Discussion by margarizaldi** [filamentphp/filament#5353] [Ideas]
I think the _required_ field indicator style is too "traditional" for modern look of filament, and sometimes looks bad, and this is the only view distraction I feel so far...

![image](https://user-images.githubusercontent.com/26832856/209797775-37901f87-b999-47e9-b7e2-f9125d3705db.png)

What if we have some additional methods like:
```php
TextInput::make('name')
    ->required()
    ->requiredIndicatorColor('primary')
    ->requiredIndicatorText('Required'),
// and also ->hideRequiredIndicator() to completely hide it
```

and it becomes:

![image](https://user-images.githubusercontent.com/26832856/209798304-406ca1a6-80f1-4fbf-8bea-a7d4e6ff76dd.png)

Should looks better...

**Comment by danharrin** [comment]
I don't agree that it looks better... but you can probably do this with CSS `::after`

**Reply by margarizaldi** [reply]
Sure
